cookiecutter.json

```diff
@@ -1,11 +1,12 @@
 {
     "target_bundle": [
         "Adafruit",
-        "Community"
+        "Community",
+        "CircuitPython Org"
     ],
-    "github_user": "{%- if cookiecutter.target_bundle == 'Adafruit' -%} adafruit {%- endif -%}",
+    "github_user": "{%- if cookiecutter.target_bundle == 'Adafruit' -%} adafruit {%- elif cookiecutter.target_bundle == 'CircuitPython Org' -%} circuitpython {%- endif -%}",
```

Comment:
Should this be the proper noun CircuitPython? (same below)
It is used in the License files.

@tannewt Follow-Up Question. Is there a CircuitPython legal entity? I thought it was held by Adafruit.

Reply:
Good question. I used lowercase to match the lower case `adafruit` value for the main bundle and because as I understand it the github username for the organization is lowercase `github.com/circuitpython` however I do notice now that `github.com/CircuitPython` does also resolve and leads to the same page. I'm not sure which one would be correct in this context. If there is no specific reason that we need to use the lowercase one though I am favor of the proper noun variant.

Reply:
I was  referring to the Copyright sections where company name is used without formatting.

# SPDX-FileCopyrightText: Copyright (c) {% now 'utc', '%Y' %} {{ cookiecutter.author_name }}{% if cookiecutter.company %} for {{ cookiecutter.company }}{% endif %}

Reply:
Ah I see, the `cookiecutter.company`. I've updated that in the latest commit to `CircuitPython Organization`

Thanks for catching that.